client/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+<!--    <include resource="org/springframework/boot/logging/logback/default.xml"/>-->
```

Comment:
I need to include base.xml or nothing is logged.
`<include resource="org/springframework/boot/logging/logback/base.xml"/>`